web/vite.config.ts

```diff
@@ -218,16 +218,16 @@ export default {
     react(),
     EnvironmentPlugin("all"),
     pluginRewriteAll(),
-    {
-      name: "singleHMR",
-      handleHotUpdate({ modules }) {
-        modules.map((m) => {
-          m.importedModules = new Set();
-          m.importers = new Set();
-        });
-        return modules;
-      },
-    },
+    // {
+    //   name: "singleHMR",
+    //   handleHotUpdate({ modules }) {
+    //     modules.map((m) => {
+    //       m.importedModules = new Set();
+    //       m.importers = new Set();
+    //     });
+    //     return modules;
+    //   },
+    // },
```

Comment:
Why it is necessary to be commented? We can manage it by another way to don't need to be commented?